src/components/StructuredNavigation/NavUtils/TreeNode.js

```diff
@@ -125,15 +125,15 @@ const TreeNode = ({
   const ariaLabel = useMemo(() => {
     if (isPlaylist) {
       return isEmpty
-        ? `Restricted playlist item ${itemIndex} of ${sectionCount}, with label ${label} starts a ${CANVAS_MESSAGE_TIMEOUT / 1000} 
+        ? `Restricted playlist item ${itemIndex}${label} starts a ${CANVAS_MESSAGE_TIMEOUT / 1000} 
           second timer to auto-advance to next playlist item`
-        : `Playlist item ${itemIndex} of ${sectionCount}, with label ${label} starting at ${screenReaderTime}`;
+        : `Playlist item ${itemIndex}${label} ${duration} starting at ${screenReaderTime}`;
     } else if (isSection) {
       return id != undefined
-        ? `Load media for Canvas ${itemIndex} of ${sectionCount}`
-        : isRoot ? `Table of contents for ${label}` : `Section for Canvas ${itemIndex} of ${sectionCount} labelled ${label}`;
+        ? `Load media for Canvas ${itemIndex}${label}${duration}`
+        : isRoot ? `Table of contents for ${label}${duration}` : `Section for Canvas ${itemIndex}${label}${duration}`;
```

Comment:
Do these read okay or does there need to be spaces added?
```suggestion
        ? `Load media for Canvas ${itemIndex}${label} ${duration}`
        : isRoot ? `Table of contents for ${label}${duration}` : `Section for Canvas ${itemIndex}${label} ${duration}`;
```